communicating.md

```diff
@@ -0,0 +1,31 @@
+# Communication
+
+Communication within the TiDB community abides by the [Contributor Covenant
+Code of Conduct](./CODE_OF_CONDUCT.md). You can reach out to the community
+members via any one of the following ways:
+
+## Slack
+
+[https://pingcap.com/tidbslack](https://pingcap.com/tidbslack)
+
+## Social Media
+
+### Blog
+
+The TiDB Blog is a great resource pool to find content! We post the latest TiDB
+news including new releases and technical articles there. Check out this page:
+[blog](https://pingcap.com/blog) or [blog-cn](https://pingcap.com/blog-cn)
+
+## TiDB Weekly
+
+[TiDB Weekly](https://pingcap.com/weekly) highlights content from the community
+along with the latest TiDB weekly update & news. You can read the TiDB Weekly
+archives to catch up on all of the latest TiDB updates and hacks.
```

Comment:
> TiDB Weekly archives 

link?

Reply:
actually https://pingcap.com/weekly records both the current weekly report and all the archives. Is it ok to use the same link for `TiDB Weekly archives`? I think maybe we should specifies that readers can find all the archives on the same link?

Reply:
Think it should be Ok if users can find them right in the same link